src/services/qix/index.js

```diff
@@ -257,6 +259,9 @@ export default class Qix {
         config.session.host = 'localhost';
       }
     }
+    if (typeof config.secure === 'undefined') {
```

Comment:
There are not whitespace before/after the other if clauses, but I will change to it